vendor.mod

```diff
@@ -86,6 +86,7 @@ require (
 	github.com/sirupsen/logrus v1.9.3
 	github.com/spf13/cobra v1.8.0
 	github.com/spf13/pflag v1.0.5
+	github.com/tonistiigi/fsutil v0.0.0-20230825212630-f09800878302
```

Comment:
There is an issue with fsutil atm that needs to be fixed but only client-side afaik so should not be an issue here. See https://github.com/docker/compose/pull/11426